vital/types/metadata.cxx

```diff
@@ -187,6 +186,21 @@ ::add( std::unique_ptr< metadata_item >&& item )
 #endif
 }
 
+void
```

Comment:
Semi-pedantic:
```suggestion
// ----------------------------------------------------------------------------
void
```

(This file is a mess, consistency-wise, but we can at least DTRT for new code. That said, it might be nice to clean this up, but maybe in a separate PR.)